### ISSUE TITLE
Pushing new markers live

### ISSUE THREAD

**Issue by catanab** [SINTEF-9012/PruneCluster#122]
As I am in a situation where the payload from the service is too big to be swallowed by the browser at once, I was thinking to have a pagination in service that will provide the payload in multiple steps.

My question here, is it possible to add more markers on the fly? I've already noticed that the data can be already updated live.

Thanks in advance.

**Comment by yellowiscool** [comment]
Yes, it is possible and one of the main feature of this library. I want to
mention that each time you add markers, all the markers are sorted. The
native sort algorithm is fast but it might start to be problematic after
100 000 markers (less on mobile). So you shouldn't have a too small
pagination. By curiosity, how many markers do you plan to display ?

Le 10 juin 2016 5:40 PM, "catanab" <notifications@github.com> a écrit :
>
> As I am in a situation where the payload from the service is too big to
be swallowed by the browser at once, I was thinking to have a pagination in
service that will provide the payload in multiple steps.
>
> My question here, is it possible to add more markers on the fly? I've
already noticed that the data can be already updated live.

**Comment by catanab** [comment]
thanks @yellowiscool for the fast response. the limit that I do have at this point is exactly 100k points, but might increase in the future. 
Is it possible to give an example how to add those on the fly? just to make sure I'm going to use the right methods.

Thanks.

**Comment by yellowiscool** [comment]
Use the `RegisterMarker` method to register each new marker, and when you are done, call  `ProcessView`.

**Comment by kasparsklavins** [comment]
I'm displaying over 200k points with a pagination step of 5k per request without many problems. There is noticeable jittering but it's bearable.
The points serverside, but not entirely sure if there is any performance gain from this in the browser.
